index.html

```diff
@@ -120,82 +132,89 @@ <h2 class="projtxt">Project Name Goes Here</h2>
-            <h1 class="mtitle" >Get started</h1>
-        </div>
-        <div class="maintxt">
-            <p class="mtext" >Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
-        </div>
-        <form action="https://formspree.io/f/mayvvvnd" method="POST">
-            <ul class="contactform">
-                <li>
-                <label for="fname">Your name</label>
-                    <input type="text" name="name" id="fname" placeholder="Jhon Doe" required="" maxlength="30" pattern="([A-ZΆ-ΫÀ-ÖØ-Þ][A-ZΆ-ΫÀ-ÖØ-Þa-zά-ώß-öø-ÿ]{1,19} ?){1,10}">
-                </li>
-                <li>
-                <label for="email-address">Your email</label>
-                    <input type="email" name="email" id="email-address" placeholder="email@domain.com" required="" pattern="(?!(^[.-].*|[^@]*[.-]@|.*\.{2,}.*)|^.{254}.)([a-zA-Z0-9!#$%&'*+\/=?^_`{|}~.-]+@)(?!-.*|.*-\.)([a-zA-Z0-9-]{1,63}\.)+[a-zA-Z]{2,15}">
-                </li>
-                <li>
-                <textarea rows="4" name="message" id="message" placeholder="Enter your message here..." required="" maxlength="500"></textarea>
-                </li>
-            </ul>
-            <button class="collabbutton" type="submit">Start collaboration</button>
-        </form>
+    <section class="contact" id="contact">
+        <div class="topimg"></div>
+        <div class="contactctn">
+            <div class="maintitle">
+                <h1 class="mtitle" >Get started</h1>
+            </div>
+            <div class="maintxt">
+                <p class="mtext" >Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
+            </div>
+            <form action="https://formspree.io/f/mayvvvnd" method="POST">
+                <ul class="contactform">
+                    <li>
+                    <label for="fname">Your name</label>
+                        <input type="text" name="name" id="fname" placeholder="Jhon Doe" required="" maxlength="30" pattern="([A-ZΆ-ΫÀ-ÖØ-Þ][A-ZΆ-ΫÀ-ÖØ-Þa-zά-ώß-öø-ÿ]{1,19} ?){1,10}">
+                    </li>
+                    <li>
+                    <label for="email-address">Your email</label>
+                        <input type="email" name="email" id="email-address" placeholder="email@domain.com" required="" pattern="(?!(^[.-].*|[^@]*[.-]@|.*\.{2,}.*)|^.{254}.)([a-zA-Z0-9!#$%&'*+\/=?^_`{|}~.-]+@)(?!-.*|.*-\.)([a-zA-Z0-9-]{1,63}\.)+[a-zA-Z]{2,15}">
+                    </li>
+                    <li>
+                    <textarea rows="4" name="message" id="message" placeholder="Enter your message here..." required="" maxlength="500"></textarea>
+                    </li>
+                </ul>
+                <button class="collabbutton" type="submit">Start collaboration</button>
+            </form>
+        </div>
+        <div class="bottomimg"></div>
     </section>
     <footer class="footer">
```

Comment:
Your footer doesn't quite look like the one from the template design.

Here's your footer:

<img width="1440" alt="Screen Shot 2022-06-03 at 13 20 57" src="https://user-images.githubusercontent.com/82426682/171923427-da2bac39-8741-4f76-83aa-03e479d9435a.png">

Here's the template design footer:

<img width="538" alt="Screen Shot 2022-06-03 at 13 21 23" src="https://user-images.githubusercontent.com/82426682/171923491-bdae5506-054d-4988-ab12-72e9ad095072.png">

You should fix the margins and the space between the social media icons. Kindly do so.